wiki/Music_theory/Tempo/de.md

```diff
@@ -0,0 +1,39 @@
+---
+tags:
+  - BPM
+  - tempo
+  - beats per minute
+  - Beats pro Minute
+---
+
+# Tempo
+
+*Siehe auch: [Timing](/wiki/Beatmapping/Timing)*
+
+::: Infobox
+![Screenshot der BPM-Anzeige auf der Beatmap-Informationsseite](img/beatmap-info-DE.png "Das Tempo des Songs (181) ist in der oberen Ecke der Beatmap-Informationsseite sichtbar")
+:::
+
+Das **Tempo** bezieht sich auf die Geschwindigkeit des Songs. Normalerweise wird das Tempo in **Beats pro Minute** (***BPM***) gemessen, also die Anzahl an [ganzen Noten](/wiki/Music_theory/Beat) in einer Minute des Songs. Beispielsweise bedeutet ein Tempo von 60 Beats pro Minute ein Beat pro Sekunde, während ein Tempo von 120 Beats pro Minute doppelt so schnell ist und zwei Beats je Sekunde entspricht. Das Tempo wirkt sich auf viele Aspekte des Spiels aus, wie zum Beispiel die [Slider-Geschwindigkeit](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) oder die [Kiai-Zeit](/wiki/Gameplay/Kiai_time) <!--strobing-->. Beatmap-Informationsseiten geben das Tempo neben einem [Metronom](https://de.wikipedia.org/wiki/Metronom) an, ein von Musikern verwendetes Gerät, das "durch akustische Impulse in gleichmäßigen Zeitintervallen ein Tempo vorgibt"[^metronom].
```

Comment:
I honestly don't know how to handle `strobing`/`strobing`. The closest thing I could think of was `ruckeln`/`abgehackt`.

See [Jerkiness](https://en.wikipedia.org/wiki/Strobbing).